main.go

```diff
@@ -66,8 +66,40 @@ func main() {
 	outputFile := flag.String("o", "", "optional output to file. Default is stdout")
 	var skipDependencies flagsSet = map[string]bool{}
 	flag.Var(&skipDependencies, "skip-dependency", "Dependency to skip generating wag clients. Can be added mulitple times e.g. -skip-dependency a -skip-dependency b")
+	overrideDependenciesString := flag.String("d", "", "Dependency name to override. You can provide multiple dependencies in the format dep1:replacementDep1,dep2:replacementDep2,...")
 	flag.Parse()
 
+	// parsing through the list of overrides to make an original:new string map
+	var overrideDependenciesList []string
+	if len(*overrideDependenciesString) > 3 {
```

Comment:
I don't think the length of string matters here

Reply:
oh i just put that there to verify if the user followed the `dependency:dependencyReplacement` format